great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py

```diff
@@ -141,50 +143,51 @@ def _atomic_prescriptive_template(
         configuration: Optional[ExpectationConfiguration] = None,
         result: Optional[ExpectationValidationResult] = None,
         runtime_configuration: Optional[dict] = None,
-        **kwargs,
-    ):
-        runtime_configuration = runtime_configuration or {}
-        include_column_name = (
-            False if runtime_configuration.get("include_column_name") is False else True
+    ) -> Tuple[str, dict, Union[dict, None]]:
+        renderer_configuration = RendererConfiguration(
```

Comment:
In the near-future state (once we update the superclass `Expectation`) these atomic "template" methods will be passed an already instantiated `RendererConfiguration` instead of the current params and then immediate instantiation of a `RendererConfiguration`. This is a private method and would not be a breaking change. It is also possible this will become a `staticmethod` if the super can also become one.